capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/addTerm.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
Don't use AngularJs here. Drupal has build in functionality to handle this type of functionality.